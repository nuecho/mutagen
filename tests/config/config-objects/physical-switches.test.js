cfgObjectTests("physicalSwitches", "physical-switches-config.json", {checkInitialExport: false});
