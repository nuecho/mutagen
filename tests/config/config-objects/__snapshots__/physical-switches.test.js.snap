// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen physicalSwitches config import-export] should create the physicalSwitches: the physicalSwitches have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ physicalswitch => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen physicalSwitches config import-export] should throw exception when new physicalSwitches to import miss mandatory properties physicalSwitches: exception has been thrown 1`] = `
Object {
  "stderr": "Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
	Mandatory properties [type] not set in physicalswitch 'emptyCfgObjectTest'
",
  "stdout": "",
}
`;

exports[`[mutagen physicalSwitches config import-export] should update the physicalSwitches: the physicalSwitches have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ physicalswitch => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
